Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2713 -- Encode hash marks and other characters in `content-search` endpoint

Resolves [EREGCSC-2713](https://jiraent.cms.gov/browse/EREGCSC-2713)

**Description**

If you search for something with a # in the query (from the combined search page or from the subjects page), you get a few unexpected effects:

- In the results, the “related regulation citations” show as “none”
- In the results, the category and subcategory labels are missing ("Subregulatory Guidance" etc)
- Anything after # in the query seems to be ignored
- If you're on the subjects page and run this kind of query, the filters don't work as expected. For example, if I uncheck "public", public items are still displayed. 

**This pull request changes:**

- TBD

**Steps to manually verify this change:**

1. Green check marks
2. Search for "`SMDL #12-002`" on Subjects page and Combined Search page on [experimental deployment](https://2sgg65154h.execute-api.us-east-1.amazonaws.com/dev1337)
   - On the Subjects page, ensure you can further filter by category and subject.  [Example](https://2sgg65154h.execute-api.us-east-1.amazonaws.com/dev1337/subjects?q=SMDL+%2312-002&subjects=63&categories=5)
